test/test_flask_history.py

```diff
@@ -0,0 +1,44 @@
+import pytest
```

Comment:
These tests are really pointless. You're checking that the ctfd container exists? Why?

If you want a test, make it a functionality test. Check that you can run `dojo flask`, enter a command, and have a resulting data/ctfd-ipython/profile_default/history.sqlite file created.